src/css/button.scss

```diff
@@ -5,3 +5,47 @@ a.button {
 a.button-primary {
   @extend .btn-primary;
 }
+
+a.button {
+  display: flex;
```

Comment:
@maymax777, As I mentioned in Slack channel, we can customize the button style based on bootstrap like as following:
```
a.button {
  @extend .btn;
  ... // additional styles here
  &-primary {
    @extend .btn-primary;
    ... // additional styles here
  }
  &-secondary {
    @extend .btn-secondary;
    ... // additional styles here
  }
  &-full-width {
    width: 100%;
  }
  &-sm {
    width: 120px;
    height: 54px;
  }
}
```